client/app/src/main/java/io/mosip/registration/app/util/JobServiceHelper.java

```diff
@@ -72,8 +86,37 @@ public Class<?> getJobServiceImplClass(String jobAPIName) {
         switch (jobAPIName) {
             case "packetSyncStatusJob":
                 return PacketStatusSyncJob.class;
+            case "synchConfigDataJob":
+                return ConfigDataSyncJob.class;
             default:
                 return null;
         }
     }
+
+    public List<SyncJobDef> getAllSyncJobDefList(){
+        return packetService.getAllSyncJobDefList();
+    }
+
+    public String getLastSyncTime(int jobId){
+        long lastSyncTimeSeconds = jobTransactionService.getLastSyncTime(jobId);
+        String lastSync = "NA";
```

Comment:
Add these constants in the strings.xml under values to support internationalization.